server/protocols/webadmin/webadmin-cassandra-data/src/main/java/org/apache/james/webadmin/dto/CassandraActionMappings.java

```diff
@@ -0,0 +1,43 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.webadmin.dto;
+
+import com.google.common.base.Preconditions;
+
+public class CassandraActionMappings {
+    public enum Action {
+        solveInconsistencies
+    }
+
+    public static CassandraActionMappings parse(String action) {
+        Preconditions.checkNotNull(action, "Action is mandatory");
+        return new CassandraActionMappings(Action.valueOf(action));
```

Comment:
what is the result if I pass `a non of any action values` string to this method? I expect an error with a meaning message

Reply:
please implement a test for this method